CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(QMidi)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} "C:\\Qt\\5.15.0\\msvc2019_64\\")
+
+find_package(Qt5 COMPONENTS Core)
+
+# Common QMidi source files
+set(SOURCES
+"${PROJECT_SOURCE_DIR}/src/QMidiOut.cpp"
+"${PROJECT_SOURCE_DIR}/src/QMidiFile.cpp"
+"${PROJECT_SOURCE_DIR}/src/QMidiIn.cpp"
+)
+
+# Platform specific QMidi source files
+if(WIN32)
+    set(SOURCES ${SOURCES}
+        "${PROJECT_SOURCE_DIR}/src/OS/QMidi_Win32.cpp"
+    )
+elseif(UNIX)
+    set(SOURCES ${SOURCES}
+        "${PROJECT_SOURCE_DIR}/src/OS/QMidi_ALSA.cpp"
+    )
+elseif(HAIKU)
+    set(SOURCES ${SOURCES}
+    "${PROJECT_SOURCE_DIR}/src/OS/QMidi_Haiku.cpp"
+    )
+elseif(APPLE)
+    set(SOURCES ${SOURCES}
+        "${PROJECT_SOURCE_DIR}/src/OS/QMidi_CoreMidi.cpp"
+    )
+endif()
+
+# Make sure all source files are present to avoid obscure compiler errors.
```

Comment:
This should already be done by CMake and so should not be necessary.

Reply:
Done.